### PR TITLE
chore: remove kuttl deletion step

### PR DESCRIPTION
**Pull request by realshuting**

### CODE REVIEW

File: other/rec-req/require-base-image/99-delete.yaml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-apiVersion: kuttl.dev/v1beta1
-kind: TestStep
-delete:
-- apiVersion: kyverno.io/v1
-  kind: ClusterPolicy
-  name: require-base-image
-commands:
-- command: kubectl delete all --all --force --grace-period=0 -n require-base-image-ns
```
Comment (realshuting) [inline other/rec-req/require-base-image/99-delete.yaml:8]: Is the force deletion still required?

Comment (realshuting) [reply other/rec-req/require-base-image/99-delete.yaml:8]: It cased a timeout issue [here](https://github.com/kyverno/policies/actions/runs/6170604599/job/16747199526?pr=750) which led to a flaky test.

Comment (chipzoller) [reply other/rec-req/require-base-image/99-delete.yaml:8]: It's still needed in some very specific cases, but looking through this test case I don't think it's needed in this one.

**Review by chipzoller — APPROVED** [review]

**Comment by chipzoller** [comment]
Let's confirm kuttl finishes successfully on this run.

**Review by chipzoller — CHANGES_REQUESTED** [review]
We must retain this deletion step because the policy is created in a non-conventional way which sets `Enforce` mode using `sed`.

**Comment by realshuting** [comment]
Seems like this change caused more failures. 

The cleanup task caused timeout, can we do them in separated steps? i.e., delete the pod first, wait 3s then delete the namespace?

**Comment by chipzoller** [comment]
Yes, the cleanups can be done in different tasks, but we still have to delete the policy which is why this was causing failures.

**Pull request closed without merging**